### ISSUE TITLE
Comments not ignoring selectors with a pseudo class on them

### ISSUE THREAD

**Issue by mikemellor11** [uncss/uncss#373]
Fixture

```html
<div class="pseudo">
</div>
```
```sass
/* uncss:ignore start */
.pseudo .notInDom{
   margin: 0;
}

.pseudo:hover .notInDom{
   margin: 0;
}
*/ uncss:ignore end */
```

Result
```sass
/* uncss:ignore start */
.pseudo .notInDom{
   margin: 0;
}
*/ uncss:ignore end */
```

Seems that the ignore tags aren't being respected when it comes to selectors with pseudo classes applied. It also the same for the /* uncss:ignore */ one off comment as well.

I've created a failing test case and started a pull request. If there's any pointers where to start i'll look at seeing if i can fix this tomorrow.

#372

**Comment by RyanZim** [comment]
I think https://github.com/uncss/uncss/blob/master/src/lib.js#L74 should be `if (dePseudify(ignore[i]) === selector) {`. Not 100% sure, but that's you're first pointer.

**Comment by mikemellor11** [comment]
This has been fixed since 0.17.3
